meta/main.yml

```diff
@@ -14,12 +14,12 @@ galaxy_info:
   # OS family.  This simplifies a lot of things for roles that support
   # Kali Linux, so it makes sense to force the installation of Ansible
   # 2.10 or newer.
-  min_ansible_version: 2.10
+  min_ansible_version: "2.10"
   namespace: cisagov
   platforms:
-    - name: Amazon
+    - name: Amazon Linux 2
       versions:
```

Comment:
Should we just remove this file since this repo does not contain an Ansible role?

Reply:
I'm ok with removing it or else replacing it's contents with something along the lines of:
```
# This file is intentionally left blank since this repo does not contain an Ansible role.
```

Reply:
I went with removing it in [c4d4715](https://github.com/cisagov/ansible-manage-user/pull/33/commits/c4d47150f5416d0b7cc90557e6cf2c732e587471).